src/python/pants/help/help_info_extracter.py

```diff
@@ -1,16 +1,17 @@
 # Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-from collections import namedtuple
+import inspect
+from dataclasses import dataclass
+from typing import List, Optional, Type
 
 from pants.base import deprecated
 from pants.option.option_util import is_list_option
+from pants.util.collections import Enum
```

Comment:
Hm still fails, realized why. This line should be 
```suggestion
from pants.util.collections.enum import Enum
```

Reply:
Hmm, I'm a bit confused as `from pants.util.collections.enum import Enum` doesn't resolve?

Reply:
Ah wait never mind. What happened is that we very recently removed that file and now we just use the standard library `from enum import Enum`. Sorry about the confusion!

Reply:
Makes sense - no worries! Shall we leave as is for now?

Reply:
No, it does not work as is. You'll need to use `from enum import Enum`.

Try running `./pants help native-build-step` as a manual test that everything works correctly.